.gitignore

```diff
@@ -5,3 +5,10 @@ dictionary.dic
 
 # python virtualenv
 venv
+
+# Emacs
```

Comment:
Do we need these?  If so, would it make sense to have a separate PR for them?

Reply:
I use emacs so it is something that helps me prevent staging auto-save files.

>would it make sense to have a separate PR for them?

Generally, the reason for splitting things up into separate changesets (PRs) is so that if there is a functional regression or bug in a software codebase, you can selectively revert individual changes in order to find which code change introduced the issue. This is a documentation repository, so splitting this out would be organization for organization's sake. I don't see a reason to split it out and so will not unless a compelling reason is made.

Reply:
OK.  The main reason I would have split it up into a separate PR is the governance model -- this seems like it could move quickly under the rules for trivial changes while the rest of the PR may turn out to be a much heavier lift.  However, I am fine with leaving it in and seeing how it goes.